api/core/time/time.isItRightTime.js

```diff
@@ -0,0 +1,56 @@
+const moment = require('moment');
+const Promise = require('bluebird');
+
+/**
+ * @public
+ * @description This function return true or false is it the right time
+ * @name gladys.time.isItRightTime
+ * @param {Object} option
+ * @param {string} option.start Beginning of the interval for validation (HH:MM:SS)
+ * @param {string} option.end End of the interval for validation (HH:MM:SS)
+ * @returns {Time} time
+ * 
+ * @example
+ * var option = {
+ *	start: '10:00:00',
+ *	end: '15:00:00'
+ * };
+ * 
+ * gladys.time.isItRightTime(option)
+ *      .then(function(isItTime){
+ *          if(isItTime){
+ *              //The time is within the given range.
+ *          }else{
+ *              //The time is not in the given range.
+ *          }
+ *      });
+ */
+
+module.exports = function isItRightTime(options){
+
+    var now = new Date();
+
+    var arrayOfSart = options.start.split(":");
+    var arrayOfEnd = options.end.split(":");
+
+    // default start time 00:00:00
+    var date = new Date();
```

Comment:
Consider renaming the variable to "startDate", more explicit

Reply:
it's not my part that I prefer to give explicit names to variables lol
but you already noticed